packages/core/useDeviceOrientation/index.ts

```diff
@@ -1,20 +1,24 @@
 /* this implementation is original ported from https://github.com/logaretm/vue-use-web by Abdelrahman Awad */
 
+import { isClient } from '@vueuse/shared'
 import { ref, Ref } from 'vue-demi'
 import { useEventListener } from '../useEventListener'
 
-export function useDeviceOrientation() {
+export function useDeviceOrientation(target: Window = window) {
```

Comment:
```suggestion
export function useDeviceOrientation(target: Window = isClient ? window : null) {
```

SSR will only work this way